README.md

```diff
@@ -20,8 +20,8 @@ The new Morpho token does not have role-based permission of functions.
 
 ### Burning tokens
 
-In the legacy Morpho token, burning tokens was made possible by transferring them to the zero address.
-Instead, in the new Morpho token, approved users can burn tokens by calling the `burn` function.
+In the legacy Morpho token, it was possible to transfer tokens to the zero address.
+This is no longer possible in the new Morpho token, but it's possible for approved users to burn tokens by calling the `burn` function.
```

Comment:
I added it to explain that instead of `transferFrom(owner, address(0), amount)` you could do `transferFrom(owner, me, amount); burn(amount)` (requiring to be approved by the owner). But this is probably useless, let's remove it :+1: